pnpm-workspace.yaml

```
@@ -0,0 +1,3 @@
+packages:
+  - "apps/*"
+  - "packages/*"
```

```suggestion
  - apps/*
  - packages/*
```
I think it actually works without quotes for some reason...minimal :sunglasses: